xla/service/gpu/transforms/layout_assignment.cc

```diff
@@ -396,13 +396,17 @@ absl::Status GpuLayoutAssignment::AddDotBackendConstraints(
   const bool is_s8_to_s32 = output_type == PrimitiveType::S32 &&
                             lhs.type == PrimitiveType::S8 &&
                             rhs.type == PrimitiveType::S8;
-  const bool is_fp8_to_fp8 = (lhs.type == PrimitiveType::F8E4M3FN ||
-                              lhs.type == PrimitiveType::F8E5M2FNUZ) &&
-                             (rhs.type == PrimitiveType::F8E4M3FN ||
-                              rhs.type == PrimitiveType::F8E5M2FNUZ);
+  const bool is_fp8 = (lhs.type == PrimitiveType::F8E4M3FN ||
+                       lhs.type == PrimitiveType::F8E5M2FNUZ) &&
+                      (rhs.type == PrimitiveType::F8E4M3FN ||
+                       rhs.type == PrimitiveType::F8E5M2FNUZ);
+
+  // Blackwell does not require the transposing of FP8 dot operands.
```

Comment:
I would phrase this "Blackwell does not benefit from the contracting dimensions being minor-most." Since there is never a requirement on transposing, just whether the contracting dim is minor-most.